client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -141,109 +140,105 @@ const MarketplaceThankYou = () => {
-									</MarketplaceHeaderTitle>
-								</MarketplaceThankyouSection>
-								<MarketplaceThankyouSection>
-									<MarketplaceHeaderTitle
-										subtitle
-										className="marketplace-thank-you__body-header wp-brand-font"
-									>
-										{ translate( 'What’s next?' ) }
-									</MarketplaceHeaderTitle>
-									<MarketplaceNextSteps>
-										<h3>{ translate( 'Plugin setup' ) }</h3>
-										<div>
-											<p>
-												{ translate(
-													'Get to know Yoast SEO and customize it, so you can hit the ground running.'
-												) }
-											</p>
-											<div>
-												<FullWidthButton
-													href={ yoastSeoPageUrl }
-													primary
-													busy={ isRequestingMenu }
-													// TODO: Menu links are not properly loading on initial request, post transfer so yoastSeoPageUrl will remain empty post transfer
-													// This should be fixed with perhaps a work around to periodically poll for the menu with various domains until it loads
-													// or maybe blocking the user from entering this flow until a domain acquires SSL
-													disabled={ !! yoastSeoPageUrl }
-												>
-													{ translate( 'Get started' ) }
-												</FullWidthButton>
-											</div>
-										</div>
-										<h3>{ translate( 'Start putting SEO to work' ) }</h3>
-										<div>
-											<p>
-												{ translate(
-													"Improve your site's performance and rank higher with a few tips."
-												) }
-											</p>
-											<div>
-												<FullWidthButton
-													href={ postsPageUrl }
-													busy={ isRequestingMenu }
-													disabled={ !! yoastSeoPageUrl }
-												>
-													{ translate( 'View posts' ) }
-												</FullWidthButton>
-											</div>
-										</div>
-									</MarketplaceNextSteps>
-								</MarketplaceThankyouSection>
-								<MarketplaceThankyouSection>
-									<MarketplaceHeaderTitle
-										subtitle
-										className="marketplace-thank-you__body-header wp-brand-font"
-									>
-										{ translate( 'How can we help?' ) }
-									</MarketplaceHeaderTitle>
+			<Masterbar>
+				<Item
+					icon="cross"
+					onClick={ () =>
+						page( `/marketplace/product/details/wordpress-seo/${ selectedSiteSlug }` )
+					}
+					tooltip={ translate( 'Go to plugin' ) }
+					tipTarget="close"
+				/>
+			</Masterbar>
+			<MarketplaceThankYouContainer className="marketplace-thank-you__container checkout-thank-you">
+				<MarketplaceThankYouHeader>
+					<img alt="yoast logo" width={ imageWidth } src={ yoastInstalledImage }></img>
+				</MarketplaceThankYouHeader>
+				<ThankYouBody>
+					<div>
+						<MarketplaceThankyouSection>
+							<MarketplaceHeaderTitle className="marketplace-thank-you__body-header wp-brand-font">
+								{ /* TODO: Change thank you message to be dynamic according to product */ }
+								{ translate( 'Yoast SEO Premium is installed' ) }
```

Comment:
If this string is going to be changed to support multiple products, I think we should either:

* Change this right now to include a placeholder, and let us provide the product name as a variable to the localized string
* Remove the `translate()` call around this string temporarily, since it won't be necessary once these are made dynamic.

Reply:
This is an old page which was implemented a few weeks back so translation might be implemented, this change is showing wierdly in this diff no idea why though. However we will be removing Yoast specific stuff from code gradually.